packages/components/app/styles/components/code-block/theme.scss

```diff
@@ -0,0 +1,156 @@
+// THEMING
+
+.hds-code-block--theme-dark {
```

Comment:
I haven't looked in details at these theming variables (and their naming); I'll do it in a next pass, for now let's assume they are OK since they do the expected job.